javascript/statements.json

```diff
@@ -87,6 +87,14 @@
           }
         }
       },
+      "await_using": {
```

Comment:
I'm never sure when something is a statement vs. an operator. await is categorized as an operator currently, should this also be put there, or not?

Reply:
An _operator_ should be more adequately called an _expression_. An expression is anything that goes into `if (...)`. A statement is anything that goes into `if (x) ...`.

Reply:
Okay, I have no strong opinion, but my initial feeling is that it would be nice if this lives together with `await`. Won't block on it, though.

Reply:
Yeah, `await` and `await using` are two distinct pieces of syntax that happen to use the same keyword. Like the `import()` expression and the `import` declaration they will live in different places.